src/Connectors/src/ConnectorBase/Services/OAuthServiceInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+namespace Steeltoe.CloudFoundry.Connector.Services
+{
+    public class OAuthServiceInfo
```

Comment:
Can we use `SSOServiceInfo` instead of adding a new class?

Reply:
SsoServiceInfo has a private setters on the class properties which didn't allow me to bind config json. Would you recommend making the property setters public on SsoServiceInfo instead? Wasn't sure if that was done for a reason I may be missing.